config/core/rootfs-configs.yaml

```diff
@@ -84,6 +84,7 @@ rootfs_configs:
       - libpcre2-8-0
       - libperl5.36
       - libpsl5
+      - liburing-dev
```

Comment:
The other library packages do not install `-dev` package (e.g. development headers).

Is it worth following exiting convention and installing the `liburing2` package directly?